nuxeo-services/nuxeo-platform-filemanager-core/src/test/java/org/nuxeo/ecm/platform/filemanager/TestFileManagerService.java

```diff
@@ -108,6 +108,14 @@ public void testDefaultCreateFromBlob() throws Exception {
         assertEquals("hello.doc", doc.getProperty("dublincore", "title"));
         assertEquals("hello.doc", doc.getProperty("file", "filename"));
         assertNotNull(doc.getProperty("file", "content"));
+
+        // let's make the same test but this time without mime-type checking
+        doc = service.createDocumentFromBlob(coreSession, input, workspace.getPathAsString(), true,
+            "test-data/hello2.doc", true);
+        assertNotNull(doc);
+        assertEquals("hello2.doc", doc.getProperty("dublincore", "title"));
+        assertEquals("hello2.doc", doc.getProperty("file", "filename"));
+        assertNotNull(doc.getProperty("file", "content"));
```

Comment:
I'm not sure to understand what is tested there... Wether `noMimeTypeCheck` is `true` or `false`, it changes nothing?
Shouldn't you check the mime-type or something like that?